SingularityService/src/test/java/com/hubspot/singularity/scheduler/SingularitySchedulerTest.java

```diff
@@ -3059,6 +3063,77 @@ public void testCronScheduleChanges() throws Exception {
   }
 
   @Test
+  public void testValidQuartzTimeZone() {
+    SingularityRequest req = new SingularityRequestBuilder(requestId, RequestType.SCHEDULED)
+        .setQuartzSchedule(Optional.of("*/1 * * * * ? 2020"))
+        .setScheduleType(Optional.of(ScheduleType.QUARTZ))
+        .setScheduleTimeZone(Optional.of("GMT"))
+        .build();
+
+    requestResource.postRequest(req);
```

Comment:
this test isn't validating anything separate and isn't asserting anything, I don't think we need it. Testing the post/validation of a valid timezone is already covered by the more detailed test you wrote further down